src/liquid/components/ld-label/test/ld-label.e2e.ts

```diff
@@ -0,0 +1,161 @@
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+
+    it('renders on the left as css component', async () => {
+      const page = await getPageWithContent(`
+        <label class="ld-label ld-label--left ld-label--m">
+          Email Address
+          <div class="ld-input">
+            <input placeholder="jane.doe@example.com" type="email">
+          </div>
+        </label>`)
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+
+    it('renders on the right', async () => {
+      const page = await getPageWithContent(`
+        <ld-label position="right" size="m">
+          Email Address
+          <ld-input placeholder="jane.doe@example.com" type="email"></ld-input>
+        </ld-label>`)
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+
+    it('renders on the right as css component', async () => {
+      const page = await getPageWithContent(`
+        <label class="ld-label ld-label--right ld-label--m">
+          Email Address
+          <div class="ld-input">
+            <input placeholder="jane.doe@example.com" type="email">
+          </div>
+        </label>`)
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+  })
+
+  describe('with html content', () => {
+    it('renders with html content', async () => {
+      const page = await getPageWithContent(`
+        <ld-label position="right">
+          <span>I love to <code style="line-height: 0">code</code>.</span>
+          <ld-checkbox></ld-checkbox>
+        </ld-label>`)
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+
+    it('renders with html content as css component', async () => {
+      const page = await getPageWithContent(`
+        <label class="ld-label ld-label--right">
+          <span>I love to <code style="line-height: 0">code</code>.</span>
+          <div class="ld-checkbox">
+            <input type="checkbox">
+            <svg
+              class="ld-checkbox__check"
+              width="14"
+              height="14"
+              fill="none"
+              viewBox="0 0 14 14"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M12 4L5.40795 10L2 6.63964"
+                stroke="currentColor"
+                stroke-width="3"
+                stroke-linecap="round"
+                stroke-linejoin="round"
+              />
+            </svg>
+            <div class="ld-checkbox__box"></div>
+          </div>
+        </label>`)
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+  })
+})
```

Comment:
As the selectors are not that easy to understand and cases can be overseen, I'd recommend adding tests that test the selectors that explicitly handle (or exclude) ld-input-message components.

Reply:
Done.